### DISCUSSION TITLE
Analog Film Emulation Software.

### DISCUSSION THREAD

**Discussion by DamianEbers** [ntsc-rs/ntsc-rs#175] [Ideas]
So, we got NTSC-RS. How about some Analog Film Software? You know. Like 35mm film...

**Comment by Asyahril395** [comment]
> So, we got NTSC-RS. How about some Analog Film Software? You know. Like 35mm film...

Like Real Film Vintage with VHS? 😀

**Reply by DamianEbers** [reply]
I really don't know...

**Comment by Degamisu** [comment]
> So, we got NTSC-RS. How about some Analog Film Software? You know. Like 35mm film...

tldr; IOS apps `Dazz cam` and `Liit`

I use an app on IOS, not sure if its for android based devices, but its called `Dazz Cam` and it provides reasonably good filters that are based off of real cameras. The VHS filter there, of course, is not accurate. But the other filters are extremely accurate. Also on IOS is `Liit` which im pretty sure is associated with `Dazz Cam`

It is a good alternative to when a _real_ film emulation software is available. Film is incredably hard to emulate since, unlike VHS's analog electric and magnetic signals that can be simulated in software, it is a big piece of emulsion that when exposed to light, it hardens. So film emulation would need a lot of calculations and an entire simulation of each grain, unless theres another way...

**Reply by DamianEbers** [reply]
Well. I use Super16. But sadly, it's only for mobile devices... I might need to contact the developer about Super16 for Desktop...

**Reply by Degamisu** [reply]
> Well. I use Super16. But sadly, it's only for mobile devices... I might need to contact the developer about Super16 for Desktop...

Thats interesting! took a look at the app, I wonder if its available for IOS. If not I'll find some alternatives. Thanks!

**Comment by Maris495** [comment]
Dehancer would be the best one for authenticity. It's a plugin available for both Windows and Mac computers.
https://www.dehancer.com/
